changes/ocd-3440.sql

```diff
@@ -0,0 +1,242 @@
+ALTER TABLE openchpl.pending_surveillance
+DROP COLUMN IF EXISTS user_permission_id CASCADE;
+
+ALTER TABLE openchpl.surveillance
+DROP COLUMN IF EXISTS user_permission_id CASCADE;
+
+-- fix a typo from a previous ticket
+UPDATE openchpl.deprecated_response_field_api 
+SET api_operation = '/surveillance-report/quarterly/{quarterlyReportId}/listings/{listingId}'
+WHERE api_operation = '/surveillance-reportquarterly/{quarterlyReportId}/listings/{listingId}'
+AND http_method = 'PUT';
+	
+-- add deprecated fields
+INSERT INTO openchpl.deprecated_response_field 
+(deprecated_api_id, response_field, removal_date, change_description, last_modified_user)
+SELECT 5, 'surveillance -> authority', '2022-04-15', 'This field is deprecated and will be removed from the response data in a future release.', -1
```

Comment:
This feature likely won't get to PROD until the 15th of November, and if we give six months for deprecation warnings, then the removal_date ought to be in May

```suggestion
SELECT 5, 'surveillance -> authority', '2022-05-15', 'This field is deprecated and will be removed from the response data in a future release.', -1
```